### ISSUE TITLE
Typedefinition for typescript

### ISSUE THREAD

**Issue by RonB** [ibrod83/nodejs-web-scraper#32]
Nice library, thank you. Could you provide the typescript types for the library?

**Comment by ibrod83** [comment]
Hey, I'll try to get to it sometime.
Perhaps you'd like to do it your self, and contribute?

**Comment by RonB** [comment]
Fair enough... i am not sure if i could, typescript is pretty new to me so
that would be a project of multiple months im afraid....:-)

Op do 9 mrt. 2023 18:20 schreef ibrod83 ***@***.***>:
